### ISSUE TITLE
Changing from Processing Shapes with Strings to Int

### ISSUE THREAD

**Issue by geurtjens** [geurtjens/CrozzleRectangleMaker#92]
There is a portion of time converting a letter in a string to a character.

To process all games:
 * `Pacman2Cluster` takes 15.725 seconds and compares with String and Char.
 * `Pacman4Cluster` takes 13.667 seconds and compares with UInt8.
 * `Pacman5Cluster` takes 13.613 seconds and compares with Int.

So how much processing is being taken up converting `String` to `Character`?

15.725 - 13.613 = 2.112 seconds to do string conversion.  So 2.112 / 15.725 * 100 = 13.4% of all processing.

In other words 15.725 /13.613 = 1.155 times faster or .155 times faster.

Normally this would be a huge improvement but our move to c++ is gaining 10x speed improvement so c++ is a better direction.

But if we wanted to keep with swift then that 13.4% of wasted CPU is worth exploring.

**Comment by geurtjens** [comment]
This was a huge saving in time.  We now have the versions made but haven't yet incorporated the faster version into the actual search yet.

**Comment by geurtjens** [comment]
There was a 1 second saving.  Haven't converted the other shapes over yet.
